cli/azd/cmd/login.go

```diff
@@ -205,8 +205,8 @@ func countTrue(elms ...bool) int {
 }
 
 func (la *loginAction) login(ctx context.Context) error {
-	if la.flags.clientID != "" || la.flags.tenantID != "" {
-		if la.flags.clientID == "" || la.flags.tenantID == "" {
+	if la.flags.clientID != "" {
+		if la.flags.tenantID == "" {
```

Comment:
Double checking: does this also work if the tenant domain is passed instead of the tenant GUID?